libs/components/datetime/src/index.ts

```diff
@@ -25,6 +25,8 @@ export { SkyFuzzyDateService } from './lib/modules/datepicker/fuzzy/fuzzy-date.s
 export { SkyTimepickerTimeFormatType } from './lib/modules/timepicker/timepicker-time-format-type';
 export { SkyTimepickerTimeOutput } from './lib/modules/timepicker/timepicker-time-output';
 export { SkyTimepickerModule } from './lib/modules/timepicker/timepicker.module';
+export { SkyTimepickerComponent } from './lib/modules/timepicker/timepicker.component';
+export { SkyTimepickerInputDirective } from './lib/modules/timepicker/timepicker.directive';
```

Comment:
Changing the exports is a breaking change and would need to wait for a major version. This is not a change we want to make at this time.